rllib/algorithms/algorithm_config.py

```diff
@@ -997,16 +997,16 @@ def build_env_to_module_connector(self, env, device=None):
         if obs_space is None and self.is_multi_agent:
             obs_space = gym.spaces.Dict(
                 {
-                    aid: env.get_observation_space(aid)
-                    for aid in env.unwrapped.possible_agents
+                    aid: env.envs[0].unwrapped.get_observation_space(aid)
```

Comment:
Cool! So this defends also against a user-wrapped (non-vectorized) MultiAgentEnv.